react-app/src/notifications/index.js

```diff
@@ -1,4 +1,60 @@
 const DAILY = 1;
 const WEEKLY = 2;
 
-export { DAILY, WEEKLY }
+const SUNDAY_WEEKDAY = 7;
+
+function _showAddScreen() { /** TODO */ }
+
+function setNotifications(shcedule = DAILY) {
+  if (!window.cordova) return;
+
+  const notification = {
+    id: shcedule,
+    title: 'BeautifulThingsApp',
```

Comment:
We should display the name of the app here, right? Why the "App" at the end and not just "BeautifulThings"?

Reply:
When I started working, I thought that the full name of the app was BeautifulThingsApp, and used this title until now.

I've just changed it to "Beautiful Things".